guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,281 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
    width = 250
-   height = 550
-   alt-text = "Showing recovery phrase one word at a time"
-   caption = "Buy bitcoin screen from the [Wallet UI Kit](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1228%3A28550)"
+   height = 541
+%}
+
+</div>
+
+## 2. The payment request
+
+The next step is present to the user an on-chain payment request so they can fund their wallet.
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/funding-request.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/funding-request@2x.png"
+   width = 250
+   height = 541
+   caption = "Show the user a payment request to fund their wallet with."
+   alt-text = "Screen showing a payment request"
    layout = "float-left-desktop -background -shadow"
 %}
 
-As noted, most exchanges offer the option to buy bitcoin using fiat currencies. Once purchased, users can transfer those funds to a separate, external wallet. This might confuse new users who don't understand the difference between exchange wallets and personal wallets.
+Present the payment request to the user with an easily scannable QR code alongside having the option to copy and share it.
 
-Bitcoin ATMs are another great option for new users to exchange cash for bitcoin in person. These machines also let users trade their bitcoin for cash, which can be useful if they are somewhere new or foreign.
+Alongside the payment request, explain to the user that a payment channel will be opened with the received amount and that a minimum amount should be sent. This amount is to cover the network fees for opening a new payment channel.
 
-Finally, new users can buy bitcoin directly from other bitcoin users such as friends, other users at a meetup, or intermediary services like [localbitcoins](https://localbitcoins.com/).
+For users who want to [manually open a channel]({{'/guide/onboarding/funding-a-wallet/#manual-channel-opening' | relative_url}}) this screen should be skippable.
 
 </div>
 
-{% include /tip/open.html label="Tip: Teach users how to transfer" icon="info" color="blue" %}
+{% include /tip/open.html label="⚡ Pay-to-open" icon="info" color="blue" %}
+
+Offering a pay-to-open Lightning service allows users to fund their wallet directly with a Lightning payment.
+
+{% include /tip/close.html %}
+
+### Network sync
+
+Sync methods like compact block filters (BIP157/158) to sync to the Bitcoin network take some time to finish. If your application uses them, you should communicate to the user that their funds will not appear until the sync is complete. You may even consider not presenting a payment request until this sync is complete. This can help prevent users from getting confused about their funds not showing up.
+
+It's recommended to start any required syncing as soon as the user opens your application to limit wait times.
+
+## 3. Sending the bitcoin
```

Comment:
```suggestion
## 3. Sending bitcoin
```

Reply:
Also, calling this `sending bitcoin` is a bit weird, from end-user perspective it's the "receiving", right? Unless. you are trying to emphasise self-sending here but I don't think we should cover that.

Reply:
Well it's the action the user taking to fund their wallet. What do you mean by self-sending? The user is sending it to them selves from whatever external source they have their bitcoin

Reply:
Yes, that's exactly what I mean, I just think calling it sending may sound confusing. Perhaps initial funding, but I am not really sure, my perception of it is that it's weird, perhaps other's don't think it is.